src/main/java/spireMapOverhaul/zones/candyland/CandyLand.java

```diff
@@ -0,0 +1,197 @@
+                return true;
+        return false;
+    }
+
+    @Override
+    public void modifyReward(RewardItem rewardItem) {
+        if(rewardItem.type == RewardItem.RewardType.RELIC && !(rewardItem.relic instanceof Mango || rewardItem.relic instanceof Pear || rewardItem.relic instanceof Strawberry || rewardItem.relic instanceof Waffle)){
+            switch(rewardItem.relic.tier){
+                case RARE:
+                    SpireAnniversary6Mod.logger.info("Replacing " + rewardItem.relic.name + " with Mango");
+                    rewardItem.relic = new Mango();
+                    rewardItem.text = rewardItem.relic.name;
+                    break;
+                case UNCOMMON:
+                    SpireAnniversary6Mod.logger.info("Replacing " + rewardItem.relic.name + " with Pear");
+                    rewardItem.relic = new Pear();
+                    rewardItem.text = rewardItem.relic.name;
+                    break;
+                case COMMON:
+                    SpireAnniversary6Mod.logger.info("Replacing " + rewardItem.relic.name + " with Strawberry");
+                    rewardItem.relic = new Strawberry();
+                    rewardItem.text = rewardItem.relic.name;
+                    break;
+                default:
+                    SpireAnniversary6Mod.logger.info("Non-standard rarity found, replacing " + rewardItem.relic.name + " with Waffle");
+                    rewardItem.relic = new Waffle();
+                    rewardItem.text = rewardItem.relic.name;
+                    break;
+            }
+        } else if(rewardItem.type == RewardItem.RewardType.POTION){
+            while(rewardItem.potion.isThrown){
+                SpireAnniversary6Mod.logger.info("Try to replace " + rewardItem.potion.name + "with drinkable potion.");
+                rewardItem.potion = AbstractDungeon.returnRandomPotion(rewardItem.potion.rarity, false);
+            }
+        }
+    }
+
+    @Override
+    public void modifyRewards(ArrayList<RewardItem> rewards) {
+        SpireAnniversary6Mod.logger.info("Turning Gold into Chocolate");
+        for(RewardItem r : (ArrayList<RewardItem>) rewards.clone()){
+            if(r.type == RewardItem.RewardType.GOLD){
+                rewards.add(new HealReward("ChocolateCoins.png", r.goldAmt));
+            }
+        }
+        rewards.removeIf(r -> r.type == RewardItem.RewardType.GOLD);
+    }
+
+    public void postAddButtons(ArrayList<AbstractCampfireOption> buttons) {
+        for (AbstractCampfireOption button : buttons){
+            if (button instanceof SmithOption){
+                button.usable = false;
+            }
+        }
+    }
+
+    public void postUseCampfireOption(AbstractCampfireOption option) {
+        if(option instanceof RestOption) {
+            AbstractDungeon.player.increaseMaxHp(5, true);
+        }
+    }
+
+    public void postCreateShopCards(ArrayList<AbstractCard> coloredCards, ArrayList<AbstractCard> colorlessCards) {
```

Comment:
You may want to prevent Gold Candy from spawning in the shop, since it will usually cost more gold that it gives.

All the times it will be a dead card aren't worth the tiny minority of times when you have Membership Card and might actually want to buy it.

Reply:
I could also increase the gold gained from Gold Candy, together with reducing the prices, it can be an investment. It would have to be more than 60 gold though, which seems like too much. Comparing Mango to Old Coin, 1 max HP is 25 gold, so to make it as strong as banana, it would have to be 50(75) gold